profile/2018-03-01-hybrid.json

```diff
@@ -0,0 +1,53 @@
+{
+    "resource-manager": {
+        "microsoft.authorization": {
+            "version": "2015-07-01",
+            "types": {
+                "locks": "2017-04-01",
+                "policyAssignments": "2016-12-01",
+                "policyDefinitions": "2016-12-01",
+                "providerOperations": "2015-07-01-preview"
+            }
+        },
+        "microsoft.compute": {
+            "version": "2017-03-30",
+            "types": {}
+        },
+        "microsoft.keyvault": {
+            "version": "2016-10-01",
+            "types": {}
+        },
+        "microsoft.network": {
+            "version": "2017-10-01",
+            "types": {}
+        },
+        "microsoft.storage": {
+            "version": "2016-01-01",
+            "types": {}
+        },
+        "microsoft.subscription": {
+            "version": "2016-06-01"
+        },
+        "microsoft.resources": {
+            "version": "2018-02-01",
+            "types": {
+                "extensionsMetadata": "2015-01-01",
+                "offers": "2015-01-01"
```

Comment:
/offers is specific to azurestack and i do not think we need to include it here since we dont have swaggers yet

Reply:
Endpoint availability and Swagger are two differents things. If `offers` exists, and then this file is an accurate description of the endpoint, it's nice to have it here. Swagger can come later, and will match that.

Reply:
They are not supposed to fail if the Swagger doesn't exist. It depends how this file is interpreted, and again, what is the purpose of this file. Is this a Stack only file? A generic file? etc.